src/bricks/transformers/jquery/JQueryReaderOptions.tsx

```diff
@@ -251,10 +251,14 @@ const SelectorCard: React.FC<{
               // Use onBlur instead of onChange for committing the change to prevent the field from being de-duped
               // if the there's an intermediate state that has the same name as another field.
               onBlur={(event: React.FocusEvent<HTMLInputElement>) => {
-                onChange({
-                  name: event.target.value,
-                  selector: selectorDefinition,
-                });
+                if (event.target.value) {
+                  onChange({
+                    name: event.target.value,
+                    selector: selectorDefinition,
+                  });
+                } else {
+                  setName(initialName);
```

Comment:
Are users allowed to have this be blank? I think it would make a better UX if we do 
```suggestion
                  setName("");
```
and add some validation error.

Reply:
The user is setting the name of a property on `@data`. It should never be blank.